<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ticket</class>
 <widget class="QDialog" name="Ticket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>5号线</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_begin">
       <property name="inputMask">
        <string>99</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
       <property name="placeholderText">
        <string>站点编号-始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_end">
       <property name="inputMask">
        <string>99</string>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
       <property name="placeholderText">
        <string>站点编号-终</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_count">
       <property name="inputMask">
        <string>99</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>票数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="edit_money">
          <property name="inputMask">
           <string>9999</string>
          </property>
          <property name="placeholderText">
           <string>金额</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_pay">
          <property name="text">
           <string>支付</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
